src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -666,6 +666,11 @@ public boolean isInGoal(int x, int y, int z) {
         if (msg.equals("damn")) {
             logDirect("daniel");
         }
+        if (msg.equals("sort")) {
```

Comment:
damn danial must be last for legacy reasons

Reply:
(((legacy))) reasons